packages/create-llama/create-app.ts

```diff
@@ -88,20 +88,41 @@ export async function createApp({
   };
 
   if (frontend) {
+    let successfulInstallation = true;
+
     // install backend
     const backendRoot = path.join(root, "backend");
     await makeDir(backendRoot);
-    await installTemplate({ ...args, root: backendRoot, backend: true });
+    try {
+      await installTemplate({
+        ...args,
+        root: backendRoot,
+        backend: true,
+      });
+    } catch (error) {
+      successfulInstallation = false;
+      console.log(error);
+    }
     // install frontend
     const frontendRoot = path.join(root, "frontend");
     await makeDir(frontendRoot);
-    await installTemplate({
-      ...args,
-      root: frontendRoot,
-      framework: "nextjs",
-      customApiPath: `http://localhost:${externalPort ?? 8000}/api/chat`,
-      backend: false,
-    });
+    try {
+      await installTemplate({
+        ...args,
+        root: frontendRoot,
+        framework: "nextjs",
+        customApiPath: `http://localhost:${externalPort ?? 8000}/api/chat`,
+        backend: false,
+      });
+    } catch (error) {
+      successfulInstallation = false;
+      console.log(error);
+    }
+    if (!successfulInstallation) {
+      console.log(red("\nOne or more installations failed. Exiting...\n"));
```

Comment:
Exists too early, all files must be generated first

Reply:
agree, we should still generate all the files (docs, git,..) that user can reference if they were failed.